api/src/main/resources/workbench.yaml

```diff
@@ -2444,6 +2516,17 @@ definitions:
         type: object
         description: The labels to be placed on the cluster. Of type Map[String,String]
 
+  FileDetail:
+    type: object
+    required:
+      - name
+      - path
+    properties:
+      name:
+        type: string
```

Comment:
Could you add a description to these properties? Is the name the filename, or something else??
Path is always gs://... etc?

Reply:
Fixed already